ohapi/api.py

```diff
@@ -173,3 +173,45 @@ def delete_file(access_token, project_member_id, base_url=OH_BASE_URL,
 # Alternate names for the same functions.
 def delete_files(*args, **kwargs):
     return delete_file(*args, **kwargs)
+
+
+def send_message_individual(subject, message, access_token, base_url=OH_BASE_URL):
+    """ 
+    send message to individual member.
+    """
+    url = urlparse.urljoin(
+        base_url, '/api/direct-sharing/project/message/?{}'.format(
+        urlparse.urlencode({'access_token': access_token})))
+    requests.post(url,data={'subject': subject,
+                            'message': message})
+
+
+def send_message_multiple(subject, message, access_token, all_members, project_member_ids, base_url=OH_BASE_URL):
+    """
+    send message to multiple members.
+    """
+    url = urlparse.urljoin(
+        base_url, '/api/direct-sharing/project/message/?{}'.format(
+        urlparse.urlencode({'access_token': access_token})))
+    if all_members  and project_member_ids:
+        raise ValueError(
+            "One (and only one) of the following must be specified: "
+            "project_members_id or all_members is set to True.")
+    requests.post(url, data={'all_members': all_members,
+                 'project_member_ids': project_member_ids,
+                 'subject': subject,
+                 'message': message})
+
+
+def messaging(subject, message, access_token, master_access_token=False, all_members=False, project_member_ids=None, base_url=OH_BASE_URL):
```

Comment:
Could you please rename the `access_token` here to `token`? Depending on which flag is set this token can be either `master` or `access`, so calling it `access_token` is confusing. 

And coming to think of it, maybe `master_access_token` should here be called `token_is_master` as it's a binary flag that just decides what the `token` should be (sorry for getting that wrong earlier myself 😞).